README.md

```diff
@@ -290,6 +325,17 @@ Interceptor called on every request. Takes Axios request config as an arg. Defau
 
 Interceptor called on every response. Takes Axios response object as an arg. Default does nothing. Pass your own function to log any desired data.
 
+#### apiAdapter (default: `new RestAdapter(configuration)`)
+
+An [`Adapter`](https://github.com/contentful/contentful-management.js/blob/2350b47053459694b21b19c71025632fe57815cc/lib/common-types.ts#L493-L495) used to
```

Comment:
```suggestion
An [`Adapter`](https://github.com/contentful/contentful-management.js/blob/2350b47053459694b21b19c71025632fe57815cc/lib/common-types.ts#L493-L495) is used to
```

Reply:
@shikaan This still needs to change because "An Adapter used to issue requests" has a wrong meaning.